Functions/Compress.cs

```diff
@@ -6,42 +6,47 @@ namespace BinaryRage.Functions
 	public static class Compress
 	{
 
-		//Compress bytes
-		public static byte[] CompressGZip(byte[] raw)
-		{
-			using (MemoryStream memory = new MemoryStream())
-			{
-				using (GZipStream gzip = new GZipStream(memory, CompressionMode.Compress, false))
-				{
-					gzip.Write(raw, 0, raw.Length);
-				}
-				return memory.ToArray();
-			}
-		}
+    //Compress bytes
+    public static byte[] CompressGZip(byte[] raw, CompressionLevel compressionLevel)
+    {
+      using (MemoryStream memory = new MemoryStream())
+      {
+        using (GZipStream gzip = new GZipStream(memory, compressionLevel, false))
```

Comment:
Do we have a reason to believe that this would be faster if we passed the real file output stream here instead of copying to memory an extra time?